openvpn.c

```diff
@@ -324,12 +324,19 @@ OnStateChange(connection_t *c, char *data)
                 c->failed_psw_attempts++;
         }
 
-        c->state = reconnecting;
-        CheckAndSetTrayIcon();
+        // We cannot interpret the state change to RECONNECTING as an indication of a restart after a prior successful connection
```

Comment:
A minor remark: Its more useful to phrase comments as applying to the new code, not to the old code. Like: We change the state to reconnecting only if there was a prior successful connection.

But I won't insist -- the code change looks good to me.

Reply:
I've update the comment, squashed and force pushed